pkgs/tools/backup/pgbackrest/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  postgresql,
+  openssl,
+  lz4,
+  bzip2,
+  libxml2,
+  zlib,
+  zstd,
+  libyaml,
+}:
```

Comment:
Using alejandra is not allowed in nixpkgs.

Reply:
Fixed. PTAL!